.github/workflows/ci.yml

```diff
@@ -21,7 +22,7 @@ jobs:
         with:
           php-version: ${{ matrix.php }}
           coverage: none
-          tools: prestissimo
+          tools: flex
```

Comment:
this is not needed right ?

Reply:
Yes. Correct. 

I just like flex more than prestissimo